pkg/verify/verify.go

```diff
@@ -348,11 +255,20 @@ func (v *ReleaseVerifier) cacheVerification(releaseDigest string, signedWith [][
 	v.signatureCache[releaseDigest] = signedWith
 }
 
-// checkFileSignatures reads signatures as "signature-1", "signature-2", etc out of a directory until
-// either the provided fn returns an error, false, or no such file exists. No more than maxSignaturesToCheck
-// will be read.
-func checkFileSignatures(ctx context.Context, dir string, maxSignaturesToCheck int, fn func(path string, signature []byte) (bool, error)) error {
-	base := filepath.Join(dir, "signature-")
+type fileStore struct {
+	directory string
+}
+
+// Signatures reads signatures as "signature-1", "signature-2", etc. out of a directory.
+func (s *fileStore) Signatures(ctx context.Context, name string, digest string, fn store.Callback) error {
+	parts := strings.SplitN(digest, ":", 3)
+	if len(parts) != 2 || len(parts[0]) == 0 || len(parts[1]) == 0 {
+		return fmt.Errorf("the provided release image digest must be of the form ALGO:HASH")
+	}
+	algo, hash := parts[0], parts[1]
+	digestPathSegment := fmt.Sprintf("%s=%s", algo, hash)
```

Comment:
Make helper. Could be used by  configmap.digestToKeyPrefix too if separator passed in.

Reply:
> Make helper.

This is just five lines.  A helper might make sense now, but seems overcomplicated for [the future][1] where the HTTP and file stores will live in separate sub-packages.

[1]: https://github.com/openshift/cluster-version-operator/pull/393#discussion_r445916218

Reply:
I'm fine with that but thought I had gotten a similar comment from someone to use a helper in a similar situation (not you necessarily).